webapi/src/main/scala/org/knora/webapi/messages/admin/responder/listsmessages/ListsMessagesADM.scala

```diff
@@ -129,8 +129,9 @@ case class CreateChildNodeApiRequestADM(parentNodeIri: IRI,
  */
 case class ChangeListInfoApiRequestADM(listIri: IRI,
                                        projectIri: IRI,
-                                       labels: Seq[StringLiteralV2],
-                                       comments: Seq[StringLiteralV2]) extends ListADMJsonProtocol {
+                                       name: Option[String] = None,
+                                       labels: Option[Seq[StringLiteralV2]] = None,
```

Comment:
If I understand correctly, the reason for the `Option` is that if you submit an empty list, it should remove all labels. Is that right? If so, could you add a test for that?

Reply:
no, if you submit `labels=[]` it will return an error. At the time being, the idea is that the update route should not be used to delete the properties. This will change later.

Reply:
So why use `Option`?

Reply:
because without it, the user needs to repeat the labels and comments even if she wants to only change the `name`

Reply:
So when `labels` and `comments` are optional, user can simply use the following request body to update only the `name` of the list without having to give/repeat the values of `labels` and `comments`
```json
   {
       "listIri": "listIri",
       "projectIri": "someprojectiri",
       "name": "another name"
  }
```

Reply:
OK, I understand, thanks.